waku/waku.go

```diff
@@ -815,7 +899,19 @@ func (w *Waku) GetSymKey(id string) ([]byte, error) {
 func (w *Waku) Subscribe(f *Filter) (string, error) {
 	s, err := w.filters.Install(f)
 	if err == nil {
-		w.updateBloomFilter(f)
+		w.settingsMu.RLock()
```

Comment:
I would create a new method and have here only something like this:
```go
w.updateSettingsForFilter(f)
```

Is there a code where we update bloom filter/topic interest after unsubscribing from a filter?